index.html

```diff
@@ -8035,6 +8037,32 @@ <h1>JSON Schema for TD Instance Validation</h1>
           "scheme"
         ]
       },
+      "autoSecurityScheme": {
+        "type": "object",
+        "properties": {
+          "@type": {
+            "$ref": "#/definitions/type_declaration"
+          },
+          "description": {
+            "$ref": "#/definitions/description"
+          },
+          "descriptions": {
+            "$ref": "#/definitions/descriptions"
+          },
+          "proxy": {
```

Comment:
do we want descriptions, proxy, etc?  To discuss in security call.  These are optional, however.  *probably* ok, if the TD wants to hide what they are doing and negotiate e.g. the proxy dynamically they can omit these.